tests/integrations/aiohttp/test_aiohttp.py

```diff
@@ -534,3 +548,197 @@ async def handler(request):
             resp.request_info.headers["baggage"]
             == "custom=value,sentry-trace_id=0123456789012345678901234567890,sentry-environment=production,sentry-release=d08ebdb9309e1b004c6f52202de58a09c2268e42,sentry-transaction=/interactions/other-dogs/new-dog,sentry-sample_rate=1.0,sentry-sampled=true"
         )
+
+
+@min_python_version(3, 8, "GraphQL aiohttp integration requires py>=3.8")
```

Comment:
we support aiohttp version 3.5 in python 3.7. Will this combination cause problems with the graphql integartion?

Reply:
Thanks for this note, it prompted me to look at this again and I've fixed it now to also work on older aiohttp versions. Don't know why I was gatekeeping it based on py version before, it was always an aiohttp compat thing, but I've worked around that too now.